pkg/controller/statefulset/stateful_set_control.go

```diff
@@ -368,6 +370,7 @@ func (ssc *defaultStatefulSetControl) updateStatefulSet(
 
 	for i := range condemned {
 		if !isHealthy(condemned[i]) {
+			unhealthyPodsCounter.WithLabelValues().Inc()
```

Comment:
is the count of unhealthy pods useful in some context ?

Reply:
If there's a lot of unhealthy pods it means something has gone awry.

To be frank this was added in the production reliability review because there is a section for detecting the health of the service. Is it necessary to add such a metric? Possibly not, what do you think from your position as sig-apps owner?

Reply:
I am not really sure we have seen any request or issues requesting a metric on this. Also condemned is really all ordinals greater than replicas, which I am not sure we care .

I am more interested in the replicas within 0 to replicas which are unhealthy.

Reply:
Ah good point on the condemned. I've done that.

If you think this metric is not useful, I'm okay with that as well. I think this conversation will justify the change to the PRR part of the KEP.